airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/retriers/chain_retrier.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from typing import List, Union
+
+import requests
+from airbyte_cdk.sources.declarative.requesters.retriers.retrier import (
+    NonRetriableResponseStatus,
+    ResponseStatus,
+    Retrier,
+    RetryResponseStatus,
+)
+
+
+class ChainRetrier(Retrier):
+    """
+    Sample config chaining 2 different retriers:
+        retrier:
+          type: "ChainRetrier"
+          retriers:
+            - retry_response_filter:
+                predicate: "{{ 'codase' in decoded_response }}"
+              backoff_strategy:
+                - type: "ConstantBackoffStrategy"
+                  backoff_time_in_seconds: 5
+            - retry_response_filter:
+                http_codes: [ 403 ]
+              backoff_strategy:
+                - type: "ConstantBackoffStrategy"
+                  backoff_time_in_seconds: 10
+    """
+
+    def __init__(self, retriers: List[Retrier]):
```

Comment:
@brianjlai, the comment above has a list of retriers. [These lines define the first one](https://github.com/airbytehq/airbyte/pull/14330/files#diff-8e3117113a7018b95d6c24de9a435cb7faa500e9d517e9a57f0ade13535c5ee2R22-R26)

The type is implied because we can default to `DefaultRetrier`